src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -481,7 +481,6 @@ export default {
 	box-sizing: border-box;
 	width: 100%;
 	min-height: $clickable-area;
-	padding-right: 4px;
```

Comment:
Removing this padding is not the solution for the misalignment. It has to stay here, otherwise the three-dots-menu and all other elements stick to the right border (which is undesired, we had a discussion about this some time ago):
![Screenshot 2021-11-10 at 15-01-23 Nextcloud Vue Style Guide](https://user-images.githubusercontent.com/2496460/141126917-a42a150c-6ac5-43ba-80b9-e03d5f5d7b3b.png)

It must only be removed for sub-appnavigationitems.

Reply:
A rule like this should fix it:
```css
.app-navigation-entry__children > .app-navigation-entry {
  padding-right: 0;
}
```
See for one level nesting:
![Screenshot 2021-11-10 at 15-22-45 Nextcloud Vue Style Guide](https://user-images.githubusercontent.com/2496460/141130409-313d0d3f-4c17-4013-9f4b-c53f1f951ca3.png)